src/tests/implementation/crypto/hash.ts

```diff
@@ -1,13 +1,12 @@
 import { IHasher } from '../../../interfaces/index.js'
-import { TextEncoder } from 'util'
-import { Blake3 } from './blake3.js'
 import { Base64 } from '../encoding/base64.js'
 
 export class Hasher implements IHasher {
   async sum(message: string): Promise<string> {
     const encoder = new TextEncoder()
     const bytes = encoder.encode(message)
-    const hash = await Blake3.sum256(bytes)
+    const hashBuffer = await crypto.subtle.digest('SHA-256', bytes)
```

Comment:
This change breaks the integration with all the other libs, I don't think we can remove the noble package quite yet. I expect blake3 will be brought into crypto.subtle in the next few years.

Reply:
ok added blake3 back in, thanks for pointing that out.